test/testng/org/openjdk/jextract/test/toolprovider/docComments/TestDocComments.java

```diff
@@ -169,6 +169,20 @@ public void testStruct2() throws IOException {
             "Setter for field: int z"));
     }
 
+    @Test
+    public void testNestedAnon() throws IOException {
+        var comments = getDocComments("structs.h", "NestedAnon.java");
+        assertEquals(comments, List.of(
+            "struct NestedAnon { struct { int l; long long h; } u; }",
+            "struct { int l; long long h; }",
+            "Getter for field: int l",
+            "Setter for field: int l",
+            "Getter for field: long long h",
+            "Setter for field: long long h",
+            "Getter for field: struct { int l; long long h; } u",
+            "Setter for field: struct { int l; long long h; } u"));
```

Comment:
Note that these last 2 comments would contain file paths without the fix.

When I was extracting Windows.h, some file paths contain `*\um\*` (which is an include directory in the Windows SDK), and this leads javac to complain about invalid unicode escape sequences.